wandb/apis/public/query_generator.py

```diff
@@ -1,5 +1,11 @@
+"""QueryGenerator is a helper object to write filters for runs."""
```

Comment:
I think this module docstring can be removed, since it's the exact same as the class docstring 4 lines below.